### PR TITLE
Revise hostname

### PR DESCRIPTION
**Pull request by yillkid**
Revise hostname for business service only.

### CODE REVIEW

**Comment by jserv** [comment]
Don't mention `localhost` as well since most developers might not install IRI locally.

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ The example below shows how easy it is to use Alfred API to issue an IOTA transa
 var request = require('request');
 
 request.post(
-  'http://node.deviceproof.org:5566/transaction/',
+  'http://tangle.biilabs.io:5566/transaction/',
```
Comment (ender503) [inline README.md]: It should be 
```suggestion
  'https://tangle-accel.biilabs.io/transaction/',
```

File: README.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ calling the prepareTransfers method and piping the prepared bundle to the sendTr
     import { composeAPI } from '@iota/core'
     
     const iota = composeAPI({
-        provider: 'http://localhost:14265' // replace with your IRI node.
+        provider: '<IOTA FULL-NODE URL:API PORT>' // replace with your IRI node.
```
Comment (jserv) [inline README.md]: Should be `<IOTA-FULLNODE:PORT>`

Comment (jserv) [reply README.md]: List `tangle.biilabs.io` for default configurations.

**Pull request merged**